test/Datadog.Trace.Tests/CorrelationIdentifierTests.cs

```diff
@@ -27,7 +27,7 @@ public void TraceIdSpanId_MatchActiveSpan()
             }
         }
 
-        [Fact(Skip = "This test is not compatible with the xUnit integration. Neither TraceId or SpanId are Zero.")]
```

Comment:
What did the history tell about this one?

Reply:
The skip was introduced here: https://github.com/DataDog/dd-trace-dotnet/pull/802
I have not found any reason why we should skip it as it works 😉 
@tonyredondo is there something that I am missing?

Reply:
It failed in Azure Pipelines. Yet, it really passed on my machine...

```
➜  Datadog.Trace.Tests git:(refactor-parsing-zipkin-settings) ✗ dotnet test

Passed!  - Failed:     0, Passed:   398, Skipped:     0, Total:   398, Duration: 22 s - /Users/rpajak/repos/opentelemetry-dotnet-instrumentation/test/Datadog.Trace.Tests/bin/Debug/netcoreapp2.1/Datadog.Trace.Tests.dll (netcoreapp2.1)

Passed!  - Failed:     0, Passed:   398, Skipped:     0, Total:   398, Duration: 22 s - /Users/rpajak/repos/opentelemetry-dotnet-instrumentation/test/Datadog.Trace.Tests/bin/Debug/netcoreapp3.0/Datadog.Trace.Tests.dll (netcoreapp3.0)

Passed!  - Failed:     0, Passed:   403, Skipped:     0, Total:   403, Duration: 26 s - /Users/rpajak/repos/opentelemetry-dotnet-instrumentation/test/Datadog.Trace.Tests/bin/Debug/netcoreapp3.1/Datadog.Trace.Tests.dll (netcoreapp3.1)
```